cli_tools/common/image/importer/api_inflater.go

```diff
@@ -101,6 +113,31 @@ func (inflater *apiInflater) Inflate() (persistentDisk, shadowTestFields, error)
 	default:
 	}
 
+	pd, ii, err := inflater.getDiskAttributes(ctx, diskName, cd, startTime)
+	if err != nil {
+		return pd, ii, err
+	}
+
+	// Calculate checksum by daisy workflow
+	inflater.logger.Debug("Started checksum calculation.")
+	ii.checksum, err = inflater.calculateChecksum(ctx, pd.uri)
+	if err != nil {
+		err = daisy.Errf("Failed to calculate checksum: %v", err)
+	}
+	return pd, ii, err
+}
+
+func (inflater *apiInflater) createDisk(diskName string) (compute.Disk, error) {
+	cd := compute.Disk{
+		Name:                diskName,
+		SourceStorageObject: inflater.request.Source.Path(),
+		GuestOsFeatures:     inflater.guestOsFeatures,
+	}
```

Comment:
This is maybe a nit, but input args for this function are mix and match of object attributes (source path, guest OS attributes) and function args (diskName). It would be cleaner if one or the other approach was used consistently. E.g. if inflater fields are used as input, then diskName can become a field on the inflater. Or, source path and guestOsFeatures are passed as args to the func.

There are other functions that do mix and match of input args like this, even though args are for the same object (disk).

Reply:
I agree that it's overall a correct thing to use the consistent way to get args. For this specific case, there can be multiple disks created, so diskName became a param.